### ISSUE TITLE
write error?

### ISSUE THREAD

**Issue by gingerbeardman** [TeamShinkansen/Hakchi2-CE#73]
I am getting this error:

```
ClovershellException
write error
   at com.clusterrr.clovershell.ClovershellConnection.writeUsb(ClovershellCommand cmd, Byte arg, Byte[] data, Int32 pos, Int32 l) in D:\C\hakchi2\Clovershell\ClovershellConnection.cs:line 447
   at com.clusterrr.clovershell.ClovershellConnection.Execute(String command, Stream stdin, Stream stdout, Stream stderr, Int32 timeout, Boolean throwOnNonZero) in D:\C\hakchi2\Clovershell\ClovershellConnection.cs:line 653
   at com.clusterrr.clovershell.ClovershellConnection.ExecuteSimple(String command, Int32 timeout, Boolean throwOnNonZero) in D:\C\hakchi2\Clovershell\ClovershellConnection.cs:line 638
   at com.clusterrr.hakchi_gui.hakchi.ShowSplashScreen() in D:\C\hakchi2\hakchi.cs:line 415
   at com.clusterrr.hakchi_gui.hakchi.ShowSplashScreen(Tasker tasker, Object syncObject) in D:\C\hakchi2\hakchi.cs:line 409
   at com.clusterrr.hakchi_gui.Tasks.Tasker.startThread() in D:\C\hakchi2\Tasks\Tasker.cs:line 377
```

**Comment by daPhie79** [comment]
what version are you trying currently?
if you're using latest, you should be able to give me much more context about the error. i've added log contents to the error report box! :)

**Comment by gingerbeardman** [comment]
when i first run hakchi2ce tonight, it prompted to update and took me to 1.2.2

i had to manually upgrade to 1.2.3, as it did not find that

this error has now gone away.

**Comment by daPhie79** [comment]
ah, sorry for the inconvenience, i released 1.2.3 silently because it still has a few known issues, but it also fixes a couple previous issues! working on 1.2.4 at the moment!

**Comment by gingerbeardman** [comment]
i kind of wish i had not updated, i now have a non-working setup now

you should disable updates from old versions until you iron out all the problems with the current version
